_posts/2018-07-27-GettingStartedWithProtobufGrpc.md

```diff
@@ -9,15 +9,16 @@ author: PrakharSrivastav
 
 There are several blog posts on the internet touching one or several shiny features that gRPC framework boasts about. There is also the [official documentation](https://grpc.io/docs/) from gRPC. It is very easy to get lost in all the jargons, terminology and the code examples. This ends up making readers confused and frustrated with a very limited understanding of what gRPC offers.
 
-The motivation behind writing this blog post is to gently introduce the gRPC framework to interested developers and the architects, skipping the more advanced details. This blog starts with setting up an initial context about the gRPC framework, followed by discussing a few of its promising features. We then highlight typical applications that can be build using gRPC and introduce the readers to a simple workflow for a small to medium scale project. Wherever possible, we have provided references to more detailed sources.
+The motivation behind writing this blog post is to gently introduce the gRPC framework to interested developers and architects, skipping the more advanced details. This blog starts by setting up an initial context about the gRPC framework, followed by discussing a few of its promising features. We then highlight typical applications that can be build using gRPC and introduce the readers to a simple workflow for a small to medium scale project. Wherever possible, we have provided references to more detailed sources.
+
+This blog focusses only on introducing gRPC as a concept and provide a convenient starting point. The code discussed in this example can be found on [github]().
 
-This blog focusses only on introducing gRPC as a concept and provide a convenient starting point. If you want to get more detailed instructions, the best place would be the [README.md](https://github.com/sysco-middleware/workshop-grpc/blob/master/README.md) file in the Github repository specified in the next section.
 
 # Repository
 
-The examples provided in this blog are loosely based on Github repository [workshop-grpc](https://github.com/sysco-middleware/workshop-grpc). Once you have completed section 2 of this blog, we encourage you to clone this repository to and follow the instructions in [README.md](https://github.com/sysco-middleware/workshop-grpc/blob/master/README.md) file to get a more detailed explanation.
+The examples provided in this blog are based on [this](https://github.com/PrakharSrivastav/grpc-protobuf-starter) Github repository. 
```

Comment:
Any chance to move repo to `sysco-middleware` org, at least for posts it is better to have it under the same org? also could you add `post-*` as prefix to be able to filter repo by name (i.e. post-grpc-protobuf-starter)?

Reply:
yes. new repo here https://github.com/sysco-middleware/post-protobuf-grpc